SUPPORT.md

```diff
@@ -0,0 +1,285 @@
+  - [Amazon Web Services (AWS)](#amazon-web-services-aws)
+  - [MongoDB](#mongodb)
+  - [Material UI](#material-ui)
+  - [React](#react)
+  - [Jest](#jest)
+  - [Golang](#golang)
+  - [Git / Github](#git--github)
+
+<!-- /TOC -->
+
+## Project Leads and Contact Info
+
+[Join our Discord Server](https://discord.gg/pPERUuv)
+
+### Anthony A (Project Organizer)
+
+Also known as "The Dude", "His Dudeness", "Duder" or "El Duderino" if you're not into the whole brevity thing.
+
+Front End Code, Documentation, and booking meetings with experts
+
+Open to peer program any issue or bugs!
+
+- Github: @tesla809
+- Discord: @Anthony A.
+
+### Carter Klein: (CTO / Back End Lead)
+
+Back End, Front End, Systems Architecture
+
+- Github: @Carter Klein
+- Discord: @Carter Klein
+
+### Brian H. Hough (UI/UX Lead)
+
+Front End, Design, Social Media Marketing
+
+- Github: @Brian H. Hough
+- Discord: @InterstellarX
+
+### Akil Hylton (Hardware Lead)
+
+Front End, Documentation, Machine Learning
+
+Open to peer program any issue or bugs!
+
+- Github: @akilhylton
+- Discord: @Akil Hylton
+
+### Nick Giangre (Analytics Lead)
+
+Data Models, Analytics, Data Science
+
+See [analytics thread / issue.](https://github.com/COVID-19-electronic-health-system/Corona-tracker/issues/51)
+
+- Github: @ngiangre
+- Discord: @NickG
+
+### Salvatore Volpe (Translations Lead)
+
+Translations, Analytics
+
+- Github: @salvolpe
+- Discord: @laseplov
+
+### Luke Lin (Discord Lead)
+
+Managing Discord, Discord Roles, Front End, Translations
+
+- Github: @lukelin1991
+- Discord: @kirbypooh
+
+### Jason Schrader (Documentation Lead)
+
+Documentation, Documentation, Documentation
+
+- Github: @whoabuddy
+- Discord: @whoabuddy
+
+## CoronaTracker Resources
+
+### Repositories
+
+__[Corona-tracker:](https://github.com/COVID-19-electronic-health-system/Corona-tracker)__ CoronaTracker is an easy-to-use and accessible progressive web application that helps you monitor your wellness and stay informed during the COVID-19 crisis, designed by an open-source community invested in public health.
+
+__[Coronalert:](https://github.com/COVID-19-electronic-health-system/Coronalert)__ CoronaTracker notification service
+
+__[Coronalytics:](https://github.com/COVID-19-electronic-health-system/Coronalytics)__ CoronaTracker analytics service
+
+__[Coronadvise:](https://github.com/COVID-19-electronic-health-system/Coronadvise)__ CoronaTracker education service
+
+__[.github](https://github.com/COVID-19-electronic-health-system/.github):__ Community health files for the [@COVID-19-electronic-health-system](https://github.com/COVID-19-electronic-health-system) organization.
+
+__[PanFLUte:](https://github.com/COVID-19-electronic-health-system/PanFLUte)__ An open source spirometer for everyone.
+
+### Technology Stack
+
+- Frontend: [React](https://github.com/facebook/react)
+- Backend: [Blockstack](https://blockstack.org/about)
+- Database: [Gaia](https://github.com/blockstack/gaia)
```

Comment:
Let's add [MongoDB](https://www.mongodb.com/cloud/atlas) to databases

Reply:
Fixed per c7bdd41, line now reads "Database: [Gaia](https://github.com/blockstack/gaia) and [MongoDB](https://www.mongodb.com/cloud/atlas)"